apigee_edge.services.yml

```diff
@@ -13,6 +13,10 @@ services:
   apigee_edge.cli:
     class: Drupal\apigee_edge\CliService
 
+  apigee_edge.edge_connection_util:
```

Comment:
IIRC we planned to use the `apigee_edge.cli` service as the source for all CLI commands that this module provides. Maybe you should add the create role command to this service and that way we do not need to worry about how we should call this new service.

Reply:
Done